src/RetriesHistogram.js

```diff
@@ -23,7 +23,7 @@ export default class RetriesHistogram extends Component {
       width = Math.round(containerRect.width - margin.left - margin.right)
     }
     const activesHeight = 72
-    const yActives = d3.scaleLinear().domain([Math.max(quota / d3.sum(flow, step => step.type === "discard"? 0 : 1), d3.max(actives, d => d.value)), 0]).range([0, activesHeight])
+    const yActives = d3.scaleLinear().domain([Math.max(quota || 1, d3.max(actives, d => d.value)), 0]).range([0, activesHeight])
```

Comment:
We can remove the `quota` property and use an `yMax` prop with more clear semantics. If `yMax` is given, use that. Otherwise use the max value of the activities.

```suggestion
    const actualYMax = this.props.yMax || d3.max(actives, d => d.value)
    const yActives = d3.scaleLinear().domain([actualYMax, 0]).range([0, activesHeight])
```